internal/core/src/indexbuilder/utils.cpp

```diff
@@ -22,10 +22,6 @@ std::vector<knowhere::IndexType>
 NM_List() {
     static std::vector<knowhere::IndexType> ret{
         knowhere::IndexEnum::INDEX_FAISS_IVFFLAT,
-#ifdef MILVUS_SUPPORT_NSG
-        knowhere::IndexEnum::INDEX_NSG,
```

Comment:
This also appeared in line46, and `milvus/internal/core/src/query/deprecated/ValidationUtil.cpp` 。suggest to clean them or, or it will be confusing

Reply:
the code is marked as RED, means it has been removed.

Reply:
line 46 removed